components/apimgt/org.wso2.carbon.apimgt.rest.api.common/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    ~ Copyright (c) 2009-2011, WSO2 Inc. (http://www.wso2.org) All Rights
```

Comment:
Shall we update the license header and add header to newly introduced class - `RestApiCommonUtil.java`